server/src/librelay/relay_server.go

```diff
@@ -668,7 +681,7 @@ func (relay *RelayServer) canRelay(from common.Address,
 }
 
 func (relay *RelayServer) validateFee(relayFee big.Int) bool {
```

Comment:
wrong test: should be `pctFee>0 || baseFee > 0`

Reply:
go server code is nearing end-of-life if we stick to the plan. Did write down the task to check JS server supports baseFee correctly